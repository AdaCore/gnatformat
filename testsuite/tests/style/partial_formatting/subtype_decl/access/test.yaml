description: SubtypeDecl formatting with partial selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 70",
    "-EL 71",
  ]
status_code: 1