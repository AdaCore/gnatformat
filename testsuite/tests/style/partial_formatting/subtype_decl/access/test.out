<CompilationUnit test.ads:19:1-72:26>
test.ads(<SubtypeDecl ["Binop_Ref"] test.ads:70:4-70:44>) - 70:1-70:44
^
   subtype Binop_Ref is not null Binop_Ptr;$

